src/pages/chainlink-nodes/configuring-nodes.mdx

```diff
@@ -0,0 +1,127 @@
+
+   ```shell
+   mkdir ~/.chainlink-toml && cd ~/.chainlink-toml
+   ```
+
+1. Write the config file from the container to your host:
+
+   ```shell
+   docker exec -it chainlink cat /home/chainlink/config.toml > ./config.toml
+   ```
+
+1. Create a `secrets.toml` file with the minimum required secrets, which are `[Database]` and `[Password]`. If you are working on a test node, you can use `?sslmode=disable` in the database `URL`:
+
+   ```shell
+   echo "[Database]
+   URL = 'postgresql://user:pass@localhost:5432/dbname'
+
+   [Password]
+   Keystore = 'keystore_pass'" > ./secrets.toml
+   ```
+
+1. Edit the `secrets.toml` file to include the secrets you need for your specific Chainlink node. See the [Node Secrets](/chainlink-nodes/v1/secrets-config) page for a full list of available config options.
+
+### Validate the configuration
+
+After you have the `config.toml` file and the `secrets.toml` files on the host system, you can validate these files using a temporary Docker container.
+
+1. Validate the configuration:
+
+   ```shell
+   docker run --name chainlink-config-validator -v ~/.chainlink-toml:/chainlink -it --rm smartcontract/chainlink:1.13.0 -config /chainlink/config.toml -secrets /chainlink/secrets.toml config validate
+   ```
+
+   You will likely see some invalid config errors. For example:
+
+   ```shell
+   Invalid configuration: invalid configuration: EVM.3.Nodes: missing: must have at least one node
+   ```
+
+1. Edit the `config.toml` file and manually correct any errors. See the [Node Config](/chainlink-nodes/v1/node-config) and [Node Secrets](/chainlink-nodes/v1/secrets-config) pages to learn which config settings are valid. For the error in this example, an EVM chain was configured with no nodes. Removing this from the config made the config valid:
+
+   ```
+   [[EVM]]
+   ChainID = '421613'
+   Enabled = false
+   Nodes = []
+   ```
+
+1. Run the `config validate` command again and make additional changes until you have a valid config message:
+
+   ```shell
+   Valid configuration.
+   ```
+
+### Restart your Chainlink node using the TOML config
+
+With your valid config and secrets files, you can migrate your Chainlink node to use the new config.
+
+1. Stop your existing Chainlink node:
+
+   ```shell
+   docker stop chainlink
+   ```
+
+1. Make a Postgres [database snapshot](https://www.postgresql.org/docs/current/backup-file.html) so you can restore your previous Chainlink node if necessary.
+
+1. Start a new Chainlink node Docker container named using the new `config.toml` and `secrets.toml` files. This example uses `chainlink-toml` as the container name:
+
+   ```shell
+   docker run --platform linux/x86_64/v8 --name chainlink-toml  -v ~/.chainlink-toml:/chainlink -it -p 6688:6688 --add-host=host.docker.internal:host-gateway smartcontract/chainlink:1.13.0 local n -config /chainlink/config.toml -secrets /chainlink/secrets.toml
```

Comment:
coming from the eng side, i'd strongly prefer `node start` to `local n` the former convey more intention and better represents what actually happens 


docker run --platform linux/x86_64/v8 --name chainlink-toml -v ~/.chainlink-toml:/chainlink -it -p 6688:6688 --add-host=host.docker.internal:host-gateway smartcontract/chainlink:1.13.0 node start -config /chainlink/config.toml -secrets /chainlink/secrets.toml

Reply:
also, the command didn't work for me on 1.13.0 i think the config and secrets flags have to be at the top level, unfortunately. this changes for 2.0.0, where they can be scoped for the `node` command

1.13:
```
docker run --platform linux/x86_64/v8 --name chainlink-toml -v ~/.chainlink-goerli:/chainlink -it -p 6688:6688 --add-host=host.docker.internal:host-gateway smartcontract/chainlink:1.13.0  -config /chainlink/config.toml -secrets /chainlink/secrets.toml node start
```

2.0.0:
the above, or, PREFERED:

```
docker run --platform linux/x86_64/v8 --name chainlink-toml -v ~/.chainlink-goerli:/chainlink -it -p 6688:6688 --add-host=host.docker.internal:host-gateway smartcontract/chainlink:1.13.0  node -config /chainlink/config.toml -secrets /chainlink/secrets.toml  start
```

Reply:
Updated to `node start` and corrected the flag order.